### ISSUE TITLE
Tooltip is not working with Ajax call

### ISSUE THREAD

**Issue by ujjavalavirus** [norcross/gravity-tooltips#24]
Hi Team,

This plugin is working superb with gravity form. Awesome UI for tooltip. All is good. But when I am calling my gravity form through ajax call this tooltip is not adding the html on form & there is no any javascript code that add this tool tip again. Please provide the solution as its urgent for me. Waiting for your reply.

**Comment by norcross** [comment]
are you referring to a multi page form? I don't quite understand the issue.

**Comment by ujjavalavirus** [comment]
Nope, We have a gravity form over the pop, Which we are adding through ajax call. So On the popup when the gravity form is loading through ajax call tooltip is getting missed out. but at the same popup,if I am adding the same gravity form direct without any ajaxcall. Tooltip is working awesome. Any clue?

**Comment by norcross** [comment]
I'd have to do some testing to see why that's the case. I'm not able to review this for a little bit as I'm currently traveling.

**Comment by norcross** [comment]
can you post a gist of how you're loading this form via Ajax?

**Comment by soulexx** [comment]
I think I know what he mean. I also have an extra plug in called ajax upload. It install a new upload field and this field doesn´t have the tooltip box to write something inside. When I installed your old version of the plugin it was working also with this new upload field because there you took the description field to generate the tooltip content.

I really like your plug in. you a great man!

**Comment by norcross** [comment]
@soulexx that's a completely separate issue. if you're having it, please open a new issue here. thanks.
